docs/readmes/basics/prerequisites.md

```diff
@@ -35,12 +35,30 @@ vagrant plugin install vagrant-vbguest
 ```
 
 If you are on MacOS, you should start Docker for Mac and increase the memory
-allocation for the Docker engine to at least 4GB (Preferences -> Advanced). 
+allocation for the Docker engine to at least 4GB (Preferences -> Advanced).
 If you are running into build/test failures with Go that report "signal killed", you
 likely need to increase Docker's allocated resources.
 
 ![Increasing docker engine resources](assets/docker-config.png)
 
+## Downloading Magma
+
+We store Magma code on [Github](https://github.com/magma/magma). There you will
```

Comment:
minor nit: This seems superfluous, we. can ask a user to checkout magma repo directly.